jobs/rocoto/gempak.sh

```diff
@@ -1,17 +1,30 @@
 #! /usr/bin/env bash
 
 source "${HOMEgfs}/ush/preamble.sh"
-
-###############################################################
-. "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
+source "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
 status=$?
-if (( status != 0 )); then exit "${status}"; fi
+(( status != 0 )) && exit "${status}"
 
 export job="gempak"
 export jobid="${job}.$$"
 
+source "${HOMEgfs}/ush/jjob_header.sh" -e "gempak" -c "base gempak"
+
+# shellcheck disable=SC2153
+fhrlst=$(echo "${FHRLST}" | sed -e 's/_/ /g; s/f/ /g; s/,/ /g')
+
+for fhr3 in ${fhrlst}; do
+    fhr=$(( 10#${fhr3} ))
+    if (( fhr > FHMAX_GFS )); then
+	echo "Nothing to process for FHR = ${fhr3}, cycle"
+	continue
+    fi
+
+    export fcsthrs="${fhr3}"
+
 # Execute the JJOB
-"${HOMEgfs}/jobs/J${RUN^^}_ATMOS_GEMPAK"
+    "${HOMEgfs}/jobs/J${RUN^^}_ATMOS_GEMPAK"
+done
```

Comment:
Based on the PR description, it seems like this should be an MPMD job or that it should be split so that only one forecast hour is run per job.  However, this seems to indicate that all forecast hours will be run in a single job.  Is the `rocoto` portion of the workflow still a work in progress?

Reply:
@DavidHuber-NOAA 
There is a parallel component for this PR that needs to be worked on to make this job run as a metatask.  Each task in the metatask will process a single forecast hour.
That exercise will be done by the workflow team as requested.